README.md

```diff
@@ -16,38 +18,29 @@ Build and push all new version of the gem:
 
 ```yaml
     steps:
-    # Setup ruby environment
     - uses: actions/checkout@v2
     - uses: ruby/setup-ruby@v1 # .ruby-version
       with:
-        bundler-cache: true # bundle install and cache
+        bundler-cache: true    # bundle install
+    - run: bundle exec rake build
 
-    - name: Build Gem
-      shell: bash
-      run: bundle exec rake build
-
-    - name: Setup GPR
-      uses: fac/ruby-gem-setup-github-packages-action@v1
+    - uses: fac/ruby-gem-setup-github-packages-action@v2
       with:
         token: ${{ secrets.github_token }}
 
-    - name: Push Gem
-      uses: fac/ruby-gem-push-action@v1
+    - uses: fac/ruby-gem-push-action@v2
+      with:
+        key: github
 ```
 
-If you want to use a different gem host or key:
-
-```yaml
-    - name: Push Gem
-      uses: fac/ruby-gem-push-action@v1
-      env:
-         gem_host: http://gems.example.com
-         gem_host_api_key: ${{ secrets.EXAMPLE_API_KEY }}
-```
+Note, that the ruby-gem-push-action, will push to the host given in the gemspec. The token needs to match. Trying to push to a different host will fail.
```

Comment:
drop the commas imo.

Also won't `fac/ruby-gem-setup-github-packages-action` set up the token so it always matches? It reads the gem host?

Reply:
I tried, I don't think I understand commas, will fix.

Sadly no on the matching, you give the key a name and tell gem push the key name. There is no matching against the host. That would make more sense, ie if the keys in ~/gem/credentials matched on host, but no! We could derive if from the host or use the first (probably only) key in the creds file, but was thinking to just keep it explicit.